src/test/OpenEXRTest/testCRgba.cpp

```diff
@@ -0,0 +1,330 @@
+//
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) Contributors to the OpenEXR Project.
+//
+
+#ifdef NDEBUG
+#    undef NDEBUG
+#endif
+
+#include "compareB44.h"
+#include "compareDwa.h"
+
+#include <IlmThread.h>
+#include <ImfArray.h>
+#include <ImfChannelList.h>
+#include <ImfFrameBuffer.h>
+#include <ImfHeader.h>
+#include <ImfOutputFile.h>
+#include <ImfCRgbaFile.h>
+#include <ImfThreading.h>
+
+#include <string>
+
+#include <assert.h>
+#include <stdio.h>
+
+using namespace OPENEXR_IMF_NAMESPACE;
+using namespace std;
+using namespace IMATH_NAMESPACE;
+
+namespace
+{
+
+void
+imfRgbaMethods ()
+{
+    //
+    // Verify that the constructors and the assignment
+    // operator for struct ImfRgba work.
+    //
+
+    ImfRgba defaultImfRgba; // Test default initialization
+    assert (defaultImfRgba.r == 0 && defaultImfRgba.g == 0 && 
```

Comment:
The default constructor for ImfRgba does *not* initialize members to 0, this line results in an uninitialized memory read. In the CI tests, some tests succeed and some fail, depending on the initialization.

But I don't think this entire function is actually needed since this basically just duplicates the tests already in `testRgba.cpp`. So the best solution is to simply remove `imfRgbaMethods()` entirely. 

I tried this test myself and it appears to pass otherwise.